dotnet/src/Experimental/Process.Abstractions/KernelProcess.cs

```diff
@@ -10,6 +10,11 @@ namespace Microsoft.SemanticKernel;
 /// </summary>
 public sealed record KernelProcess : KernelProcessStepInfo
 {
+    /// <summary>
+    /// Event raised internally for errors not handled at the step level.
+    /// </summary>
+    public const string GlobalErrorEventId = "Global.OnError";
```

Comment:
Could we namespace this similar to the End step id i.e. Microsoft.SemanticKernel.Process.Global.OnError

Reply:
Added a shared constants file for both